### PR TITLE
Add horizonslife

### PR DESCRIPTION
**Pull request by jvnazz**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/horizonslife/background.png
Comment (maaaathis) [inline minecraft_servers/horizonslife/background.png:1]: Compress this file

File: minecraft_servers/horizonslife/background@2x.png
Comment (maaaathis) [inline minecraft_servers/horizonslife/background@2x.png:1]: Compress this file

**Review by maaaathis — CHANGES_REQUESTED** [review]
The Domain `horizonslife.eu` is currently not registered at EURid:
image

**Comment by jvnazz** [comment]
> The Domain `horizonslife.eu` is currently not registered at EURid: image

Currently being set up. Will be available within the next 24 hours.

**Comment by jvnazz** [comment]
> The Domain `horizonslife.eu` is currently not registered at EURid: image

Should now be accessible.

**Comment by DoJapHD** [comment]
Server is in maintenance mode;
Reopen PR once server is publicly available

**Pull request closed without merging**